CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 -   Added: `length-zero-no-unit` autofix ([#2861](https://github.com/stylelint/stylelint/issues/2861)).
 -   Added: `selector-max-specificity` support for level 4 evaluation context pseudo-classes ([#2857](https://github.com/stylelint/stylelint/issues/2857)).
+-   Fixed: aAccept multiple spaces before comments in `selector-list-comma-newline-after` rule ([#2912](https://github.com/stylelint/stylelint/issues/2912)).
```

Comment:
~Typo here `Accept`, not `aAccept` ;)~

Reply:
Actually, can you remove this change to CHANGELOG.md entirely from this PR please.

We update CHANGELOG.md per the docs here https://github.com/stylelint/stylelint/blob/master/docs/developer-guide/pull-requests.md after the PR has been merged, it helps us avoid CHANGELOG.md merge conflicts from multiple PRs :)

Reply:
Sure I can. In `rvm` project we actually enforce it the other way around asking people to provide CHANGELOG entries :)

Reply:
We did it that way around originally, but we constantly got merge conflicts in our PRs.